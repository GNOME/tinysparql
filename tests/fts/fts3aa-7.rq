# Match contains an unfinished quote
SELECT ?o WHERE { ?o fts:match "four \"two" } order by ?o
