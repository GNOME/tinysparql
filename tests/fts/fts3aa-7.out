"http://www.example.org/test#10"
"http://www.example.org/test#11"
"http://www.example.org/test#14"
"http://www.example.org/test#15"
"http://www.example.org/test#26"
"http://www.example.org/test#27"
"http://www.example.org/test#30"
"http://www.example.org/test#31"
